### ISSUE TITLE
Mozilla Stumbler requests far more permissions than it requires

### ISSUE THREAD

**Issue by crankycoder** [mozilla/MozStumbler#1532]
No idea why this is happening, but Cyanogenmod seems to think we have access to more permissions than we define in our AndroidManfest.xml.

I'm wondering if the declaration of 

```
<uses-feature
   android:name="android.hardware.telephony"
   android:required="false" />
```

is causing this.

![2015-03-03 10 57 31](https://cloud.githubusercontent.com/assets/4829/6466238/24551490-c194-11e4-9672-d44c3c24cd5b.png)

**Comment by Djfe** [comment]
This looks like an issue in CyanogenMod to me -> you should report it

**Comment by crankycoder** [comment]
Yeah, I'm pretty sure it is a bug in CM.  I've posted in the forum, but I just found their JIRA tracker.

Bug is being tracked over here: https://jira.cyanogenmod.org/browse/BACON-3101

I'm closing this for now as permissions are fine as displayed on the google playstore and `aapt dump badging` on the apk file.

![2015-03-04 16 37 48](https://cloud.githubusercontent.com/assets/4829/6494056/06370f6e-c28d-11e4-9e9e-24b76741f05d.png)
